hicommonwealth/commonwealth
# External API GET threads and topics

## Link to Issue
Closes: #9015
Closes: #9103

## Description of Changes
- Adds GetThreads and GetTopics endpoints

## Test Plan
- Navigate to http://localhost:8080/api/v1/docs/ and test the corresponding API endpoints
- Create a thread on dydx, ensure the topic dropdown works correctly (You can see multiple topics, and you can choose one).
